unit/solvers/refinement/string_constraint_generator_valueof/get_numeric_value_from_character.cpp

```diff
@@ -18,24 +18,44 @@
 SCENARIO("get_numeric_value_from_character",
   "[core][solvers][refinement][string_constraint_generator_valueof]")
 {
-  typet char_type=unsignedbv_typet(16);
-  typet int_type=signedbv_typet(32);
-  symbol_tablet symtab;
-  namespacet ns(symtab);
-
-  REQUIRE(from_integer(0, int_type)==simplify_expr(
-    get_numeric_value_from_character(
-      from_integer('0', char_type), char_type, int_type),ns));
-  REQUIRE(from_integer(9, int_type)==simplify_expr(
-    get_numeric_value_from_character(
-      from_integer('9', char_type),
-      char_type, int_type), ns));
-  REQUIRE(from_integer(10, int_type)==simplify_expr(
-    get_numeric_value_from_character(
-      from_integer('A', char_type),
-      char_type, int_type), ns));
-  REQUIRE(from_integer(35, int_type)==simplify_expr(
-    get_numeric_value_from_character(
-      from_integer('z', char_type),
-      char_type, int_type), ns));
+  const typet char_type=unsignedbv_typet(16);
+  const typet int_type=signedbv_typet(32);
+  const symbol_tablet symtab;
+  const namespacet ns(symtab);
+
+  REQUIRE(
+    from_integer(0, int_type)==simplify_expr(
+      get_numeric_value_from_character(
+        from_integer('0', char_type), char_type, int_type),
+      ns));
+
+  REQUIRE(
+    from_integer(9, int_type)==simplify_expr(
+      get_numeric_value_from_character(
+        from_integer('9', char_type), char_type, int_type),
+      ns));
+
+  REQUIRE(
+    from_integer(10, int_type)==simplify_expr(
+      get_numeric_value_from_character(
+        from_integer('A', char_type), char_type, int_type),
+      ns));
+
+  REQUIRE(
+    from_integer(35, int_type)==simplify_expr(
+      get_numeric_value_from_character(
+        from_integer('z', char_type), char_type, int_type),
+      ns));
+
+  REQUIRE(
+    from_integer(0, int_type)==simplify_expr(
+      get_numeric_value_from_character(
+        from_integer('+', char_type), char_type, int_type),
+      ns));
+
+  REQUIRE(
+    from_integer(0, int_type)==simplify_expr(
```

Comment:
Not blocking since not changed in this PR in particular, but it would be nice to call `get_numeric_value_from_character` with some different `char_type` and `int_type`